app/repository/pg/user.go

```diff
@@ -22,6 +23,14 @@ func NewRepo(opt cfg.Options) *Repo {
 // without even knowing it that allow us to decouple our layers
 // and will make our app flexible and maintainable.
 func (r *Repo) Add(usr entities.User) error {
-	// INSERT INTO "users" ...
+	sqlStatement := `
+INSERT INTO users (first_name, last_name, email, password)
+VALUES ($1, $2, $3, $4)
+RETURNING id`
```

Comment:
I think that if our function does not return anything but an error, then the `RETURNING id`  in the request is not necessary

Reply:
There was the same comment from @Brigant , fixed that in commit above.